### PR TITLE
DTSPO-6352 Create new private endpoint

### PR DESCRIPTION
**Pull request by mokainos**

### CODE REVIEW

**Review by WillTaylor22 — APPROVED** [review]

File: privateendpoint.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    private_connection_resource_id =  module.tt-database.id
+    subresource_names              = ["postgresqlServer"]
+    is_manual_connection           = false
+  }
+
+  private_dns_zone_group {
+    name                 = "postgres-endpoint-dnszonegroup"
+    private_dns_zone_ids = ["/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/resourceGroups/core-infra-intsvc-rg/providers/Microsoft.Network/privateDnsZones/privatelink.postgres.database.azure.com"]
+  }
+    tags  = var.common_tags
```
Comment (timja) [inline privateendpoint.tf:32]: run `terraform fmt` please

File: privateendpoint.tf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+resource "azurerm_resource_group" "petrg" {
+  provider = "azurerm.pet-aks"
+  name                = "${var.product}-${var.env}-endpoint-rg"
+  location            = var.location
+
+}
+resource "azurerm_private_endpoint" "petpostgres" {
+  provider = "azurerm.pet-aks"
```
Comment (timja) [inline privateendpoint.tf:15]: where is this provider getting configured?

Comment (timja) [reply privateendpoint.tf:15]: https://github.com/hmcts/tax-tribunals-shared-infrastructure/blob/master/main.tf#L17-L21

**Review by timja — APPROVED** [review]
looks good apart from fmt being off

**Pull request merged**